src/main/java/seedu/address/model/person/Person.java

```diff
@@ -117,20 +130,23 @@ public boolean equals(Object other) {
             isEqual = isEqual && birthday.equals(otherPerson.birthday);
         }
 
+        isEqual = isEqual && tags.equals(otherPerson.tags);
+
         return isEqual;
     }
 
     @Override
     public int hashCode() {
         // use this method for custom fields hashing instead of implementing your own
-        return Objects.hash(name, phone, email, roomNumber, telegram, birthday);
+        return Objects.hash(name, phone, email, roomNumber, telegram, birthday, tags);
     }
 
     @Override
     public String toString() {
         ToStringBuilder sb = new ToStringBuilder(this);
         sb.add("name", name);
         sb.add("phone", phone);
+        sb.add("tags", tags);
```

Comment:
I'm quite curious that when is this function used, I didn't manage to find any in our current codebase (except its corresponding test function).

My main concern is the formatting issue as tags is a Set instead of a string, and it will be good if there is some visible usage to verify whether it's behaviour is as intended.

Reply:
Agreed, perhaps we could look into this and see what are the effects of this